src/pages/Playlist/component/PlaylistList.js

```diff
@@ -1,16 +1,42 @@
 import React from "react";
+import { useParams } from "react-router-dom";
 import PlaylistCard from "./PlaylistCard";
+import { useData } from "../../../context/data/videoContext";
 
 export function PlaylistList() {
```

Comment:
Name of the component could be "Playlist" only instead of PlaylistList.It's  Just more readable that way

Reply:
Yes will do that